doc/fwutil/fwutil.md

```diff
@@ -257,22 +280,55 @@ Chassis   Module   Component   Firmware               Version (current/available
                    FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date
+                   SSD         <image_path>/ssd.bin   4 / 5                                    update is required
 ```
 
-2. Modular chassis platform
+2. Non modular chassis platform when the "boot" key and value are available in `platform_components.json`
+```bash
+root@sonic:~# fwutil show updates --image=next
+Chassis   Module   Component   Firmware               Version (current/available)              Status
+--------  -------  ----------  ---------------------  ---------------------------------------  ------------------
+Chassis1  N/A      BIOS        <image_path>/bios.bin  0ACLH004_02.02.007 / 0ACLH004_02.02.010  update is required
+                   CPLD        <image_path>/cpld.bin  5 / 10                                   update is required
+                   FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date
+                   SSD         <image_path>/ssd.bin   4 / 5                                    update is required
+```
+
+3. Modular chassis platform
 ```bash
 root@sonic:~# fwutil show updates --image=next
 Chassis   Module   Component   Firmware               Version (current/available)              Status
 --------  -------  ----------  ---------------------  ---------------------------------------  ------------------
 Chassis1           BIOS        <image_path>/bios.bin  0ACLH004_02.02.007 / 0ACLH004_02.02.010  update is required
                    CPLD        <image_path>/cpld.bin  5 / 10                                   update is required
                    FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date
+                   SSD         <image_path>/ssd.bin   4 / 5                                    update is required
           Module1  CPLD        <image_path>/cpld.bin  5 / 10                                   update is required
                    FPGA        <image_path>/fpga.bin  5 / 5                                    up-to-date
 ```
 
+4. Custom FW Package when the "boot" key and value are available in `platform_components.json`
+```bash
+root@sonic:~# fwutil show updates --fw-image=fw_update.tar.gz
+Chassis   Module    Component   Firmware                       Version (current/available)              Status
+--------  -------   ----------  -----------------------------  ---------------------------------------  ------------------
+Chassis1  N/A       CPLD        <fwpackage_path>/cpld.bin      5 / 10                                   update is required
+                    SSD         <fwpackage_path>/ssd.bin       4 / 5                                    update is required
+```
+
+5. Auto-update status
+```bash
+root@sonic:~# fwutil show auto_update_status
+Firmware auto-update performed for cold reboot
+Component      Status    Info
+-------------  --------  ---------
+MSN2700/SSD    True      installed
```

Comment:
@sujinmkang This is not too much informative:
```
Status
--------
True
```
It would be much better to have something like `status=installed/updated/scheduled`:
```
Status    Info
--------  ---------------------
updated   No further actions required
scheduled Pending cold reboot
```